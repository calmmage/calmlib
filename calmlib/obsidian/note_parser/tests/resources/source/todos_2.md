- [ ] a
    - item under unchecked todo
- aa
    - item under double-letter todo
- [x] b
    - item under checked todo
- [ ] c
    - text
- d
    - [ ] todo text
- [x] e
    - [ ] text
