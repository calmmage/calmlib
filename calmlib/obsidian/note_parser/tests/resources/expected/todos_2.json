{
  "items": [
    {
      "type": "todo",
      "text": "a",
      "checked": false,
      "children": [
        {
          "type": "list_item",
          "text": "item under unchecked todo"
        }
      ]
    },
    {
      "type": "todo",
      "text": "aa",
      "children": [
        {
          "type": "list_item",
          "text": "item under double-letter todo"
        }
      ]
    },
    {
      "type": "todo",
      "text": "b",
      "checked": true,
      "children": [
        {
          "type": "list_item",
          "text": "item under checked todo"
        }
      ]
    },
    {
      "type": "todo",
      "text": "c",
      "checked": false,
      "children": [
        {
          "type": "list_item",
          "text": "text"
        }
      ]
    },
    {
      "type": "todo",
      "text": "d",
      "children": [
        {
          "type": "list_item",
          "text": "todo text",
          "checked": false
        }
      ]
    },
    {
      "type": "todo",
      "text": "e",
      "checked": true,
      "children": [
        {
          "type": "list_item",
          "text": "text",
          "checked": false
        }
      ]
    }
  ]
}